ds_judgements_public_ui/sass/includes/_homepage_cards.scss

```diff
@@ -0,0 +1,146 @@
+.homepage-cards {
+  &__container {
+    padding-top: $space-8;
+    padding-bottom: 0;
+
+    h2 {
+      display: none;
+
+      @media (max-width: $grid-breakpoint-medium) {
+        @include heading_text;
+        @include header($typography-header-h2-size);
+
+        display: contents;
+        margin-left: 0;
+        text-wrap: wrap;
+      }
+    }
+
+    @media (max-width: $grid-breakpoint-medium) {
+      margin-top: $space-4;
+      margin-bottom: $space-4;
+      padding: $space-3;
+      padding-bottom: 12.5rem;
+
+      border: 3px solid $color-yellow;
+    }
+
+    @include container;
+  }
+
+  &__two_thirds_info_cards_home {
+    display: flex;
+    flex-direction: row;
+    gap: $space-1;
+
+    margin-left: 0;
+    padding-top: $space-8;
+
+    @media (max-width: $grid-breakpoint-medium) {
+      float: left;
+      flex-direction: column;
+
+      margin-right: $space-4;
+      margin-left: $space-4;
+      padding-top: 0;
+    }
+  }
+
+  &__one_third_info_cards_home {
+    display: flex;
+    flex-direction: row;
+    gap: $space-4;
+
+    max-width: 24rem;
+    margin: $space-2;
+
+    background-color: colour-var("accent-background-light");
+    border-top: 4px solid $color-yellow;
+
+    &:hover {
+      text-decoration: none;
+      background-color: colour-var("accent-background");
+    }
+
+    @media (max-width: $grid-breakpoint-medium) {
+      height: $space-6;
+      margin-left: 0;
+      background-color: transparent;
+      border-top: none;
+
+      &:hover {
+        background-color: transparent;
+      }
+    }
+  }
+
+  &__one_third_info_cards_home_image {
+    width: 111px;
+    height: 175px;
+
+    @media (max-width: $grid-breakpoint-medium) {
+      display: none;
+    }
+
+    @media (width <= 1199px) {
+      display: none;
+    }
```

Comment:
Just looking - isn't this essentially just from mobile we want to display none? "width less then or equal to 1199px"